cmake/external/tvm.cmake

```diff
@@ -0,0 +1,35 @@
+if (onnxruntime_USE_STVM)
```

Comment:
You still need to add nuphar and all its dependencies that will be used in your build to cgmanifest.json.

Reply:
TVM dependencies are listed here: https://github.com/apache/tvm/blob/main/LICENSE#L204. Some of them are checked in the repository and the sources are difficult to trace (cma). Another comes from another dependency already listed in this file (compiler-rt / llvm, blockingconcurrentqueue.h).